### ISSUE TITLE
Support for description option for "in: :body" parameter

### ISSUE THREAD

**Issue by DominikAlberski** [rswag/rswag#403]
Currently description is not visible for parameter when "in: :body" option is set.

```ruby
parameter(
  name: :bogus,
  in: :path,
  type: :string,
  description: "descryption test"
)
```
leads to:
```yaml
parameters:
- name: bogus
  in: path
  description: descryption test
```
and:
```ruby
parameter(
  name: :params,
  in: :body,
  description: descryption test
```
```yaml
requestBody:
  # missing description key here :-(
  content:
    application/json:
      schema:
```
When added manually to yaml it is properly displayed by swagger UI

**Comment by mynnx** [comment]
@DominikAlberski let me know if that PR addresses your issues!

**Comment by DominikAlberski** [comment]
@mynnx I think it will be good, unfortunately, I no longer have access to the project where I used rswag in, so can't check it.

**Comment by jtannas** [comment]
Closing since it looks like the issue has been addressed by the PR that was merged yesterday.
New release should be relatively soon.
Thanks for contributing :)

**Comment by mscdit** [comment]
Tried to add a description for a body parameter ... without any appearance in the swagger.yaml ... has this bin fixed in rswag 2.5.1?

`describe 'Items API' do
  path '/api/v1/items' do
    put 'creates an item' do
      tags 'items'
      security [ Bearer: {} ]
      description 'Creates a new item with given key/value data attributes.'
      consumes 'application/json'
      parameter name: :blog, in: :body, description: 'test', schema: {
        type: :object,
        properties: {
          name: { type: :string },
          brand: { type: :string }
        },
        required: [ 'name', 'brand' ]
      }`
